checkly.go

```diff
@@ -45,6 +45,11 @@ func NewClient(
 	return c
 }
 
+// Set checkly account id (required when using User API keys)
+func (c *client) SetAccountId(ID string) {
```

Comment:
checkout godoc guidelines https://go.dev/blog/godoc
It should be sth. more like 
```SetAccountId sets ID on a client which is required when using User API keys.```